_pages/backup-restore.md

```diff
@@ -25,6 +25,19 @@ To take the backup:
 1. Once the backup is complete, refresh the *Backups* list to check it is complete (it may take some time) - note that the backup process is run as a Skyve *Job* and the progress and completion of the backup can be reviewed from the admin *Jobs* menu
 1. If you wish to download the backup, select the backup from the list and press *Download Backup*
 
+### Backup options
+Before making a backup, there are options found next to the *Backup* button, highlighted below:
+
+- **Content** - Check this to include content in the backup that is made
+- **Audit Log** - Check this to include the audit log that are made by the system into the backup created.
+- **Sensitivity** - While making a backup, skyve has the ability to redact certain data for security purposes. The drop-down is used to select the sensitivity level to be used during redaction. Attributes with security level that is greater than or equal to the security level selected are redacted. Below are the security levels available in ascending order in terms of sensitivity:
```

Comment:
Isn't it less than or equal?

Reply:
![image](https://github.com/user-attachments/assets/a16301d2-6f78-4683-8e95-c6fe4d42c52b)
I actually got that from the description that is there currently. I've never actually used the feature so I went by that

Reply:
So it should be less than?